readme.md

```diff
@@ -1,13 +1,85 @@
 # Navbar challenge
 
-Note: Most of the progress is in the `navbar` branch.
+## Site
 
-This repo contains the solution to this challenge https://www.notion.so/alexdan/Challenge-1-Navbar-62ae547aa708477bbd679c27a829cf6f
+Hosted using github pages [here](https://pages.github.com/)
 
-The general idea is to have a navbar like the one in this image
+## About
 
-![Navbar dribbble gif](https://cdn.dribbble.com/users/414694/screenshots/5487895/media/61c4a366fd91e33a29bdca6cfdf5f68d.gif)
+This repo holds a `Navbar` component, visually similar to the one in this dribbble:
 
-# Notes
+https://dribbble.com/shots/5487895-Tab-bar-active-animation/attachments/10896293?mode=media
 
-- Added [cypress](https://cypress.io/) tests and [github actions](https://docs.cypress.io/guides/continuous-integration/github-actions) to run them on every push.
+![Navbar gif](https://media0.giphy.com/media/v1.Y2lkPTc5MGI3NjExYjZ5MHpudjVoeHVnNTdpaDFheWRvZml3bG43M2FxdG9zNHcweWpyZiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/qmHC2pILnevhCXbwq5/giphy.gif)
+
+## Setup
+
+- Steps on how to install, run, and test. (don't forget to document the Percy key!)
+
+## Features
+
+- 🌑 / ☀️ Dark and Light theme
+- 🧱 Reusable
+- 📷 Automated visual tests with Cypress & Percy
+- ♿️ Motion a11y approach
```

Comment:
nit: this is not a "a11y approach", it's a feature for a11y. like this: "Reduced motion for A11Y" or simply "Reduced motion" (with a link pointing to an article talking about it)